test/controllers/APIControllerSpec.scala

```diff
@@ -61,11 +63,13 @@ class APIControllerSpec extends SpecBase {
 
     s"return OK (${Status.OK}) when the call returns some data and can be parsed to the correct response" in new Setup {
       when(mockETMPService.getPenaltyDataFromETMPForEnrolment(ArgumentMatchers.any())(ArgumentMatchers.any()))
-        .thenReturn(Future.successful((Some(mockETMPPayloadResponseAsModel), Right(GetETMPPayloadSuccessResponse(mockETMPPayloadResponseAsModel)))))
+        .thenReturn(Future.successful((Some(mockETMPPayloadForAPIResponseData), Right(GetETMPPayloadSuccessResponse(mockETMPPayloadForAPIResponseData)))))
       val result = controller.getSummaryDataForVRN("123456789")(fakeRequest)
       status(result) shouldBe Status.OK
-      //TODO: change data based on implementation
-      contentAsString(result) shouldBe ""
+      val apiDataToReturn: APIModel = APIModel(
+        mockETMPPayloadForAPIResponseData.pointsTotal
+      )
+      contentAsString(result) shouldBe Json.toJson(apiDataToReturn).toString()
```

Comment:
You could do contentAsJson(result) which saves you doing the toString?
```suggestion
      contentAsJson(result) shouldBe Json.toJson(apiDataToReturn)
```